### PR TITLE
Dish category feature

### PR DESCRIPTION
**Pull request by ScottyyyB**
WIP

### CODE REVIEW

File: features/step_definitions/landing_page_steps.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Given(/^there is a dish called "([^"]*)" with a price of "([^"]*)" with description of "([^"]*)" and category of "([^"]*)" in the system$/) do |product_name, cost, description, category|
+  Product.create(name: product_name, price: cost.to_i, description: description, category: category)
+end
+
+Given(/^there is a restaurant called "([^"]*)" with an owner named "([^"]*)"$/) do |name, owner|
+  Restaurant.create(name: name, owner: owner)
+end
```
Comment (tochman) [inline features/step_definitions/landing_page_steps.rb]: ?? Why is a Restaurant object created here?

File: features/step_definitions/basic_steps.rb [hunk 2]
```diff
@@ -15,3 +15,16 @@
...
+end
+
+Then(/^I would like to see "([^"]*)" with the price of "([^"]*)"$/) do |product_name, price|
+  expected_output = "#{product_name} - #{price}"
+  expect(page).to have_content expected_output
+end
+
+Then(/^I would like to see "([^"]*)" with an owner named "([^"]*)"$/) do |name, owner|
+  expected_output = "#{name} - #{owner}"
+end
```
Comment (tochman) [inline features/step_definitions/basic_steps.rb]: Where is this step being used?

File: features/list_dishes_on_landing_page.feature [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    And there is a dish called "Hamburger" with a price of "30" with description of "Great Hamburger" and category of "Main" in the system
+    And there is a dish called "Grilled Cheese" with a price of "50" with description of "Great Grilled Cheese" and category of "Dessert" in the system
+    When I visit the site
+    Then I would like to see "Hot Dog" with the price of "20 kr"
+    And I would like to see "Great Hot Dog"
+    And I would like to see "Hamburger" with the price of "30 kr"
+    And I would like to see "Great Hamburger"
+    And I would like to see "Grilled Cheese" with the price of "50 kr"
+    And I would like to see "Great Grilled Cheese"
+    Then show me the page
```
Comment (tochman) [inline features/list_dishes_on_landing_page.feature]: Please remove this step from the scenario

**Review by ScottyyyB — COMMENTED** [review]
Removed features out of scope 👍

File: features/list_dishes_on_landing_page.feature [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  As a customer/visitor
+  In order to decide on what dish to order
+  I would like to see a list of available dishes the moment the site displays
+
+  Scenario: Visitor see a list of dishes on the page
+    Given there is a dish called "Hot Dog" with a price of "20" with description of "Great Hot Dog" and category of "Starter" in the system
+    And there is a dish called "Hamburger" with a price of "30" with description of "Great Hamburger" and category of "Main" in the system
+    And there is a dish called "Grilled Cheese" with a price of "50" with description of "Great Grilled Cheese" and category of "Dessert" in the system
+    When I visit the site
+    Then I would like to see "Hot Dog" with the price of "20 kr"
```
Comment (tochman) [inline features/list_dishes_on_landing_page.feature:11]: What else?  I see that you are grouping the products in categories. Please reflect that in this test (modify the title) and/or create another scenario. Please involve your teammates in this work. We would like to see every other commit made by another developer.

Comment (ScottyyyB) [reply features/list_dishes_on_landing_page.feature:11]: Alright made a new commit with the updated Scenario which includes the fact that the dishes are grouped by category, Lisa and I will make sure to have interchanging commits when we start on the order feature. Not sure if interchanging is the correct term, but I think you know what I mean :P

**Review by tochman — APPROVED** [review]
I can merge this, but I would still like to see more extensive tests for how the dishes are actually displayed. (Grouped in categories)

**Pull request merged**